edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py

```diff
@@ -174,18 +176,44 @@ def test_authenticate_credentials_no_usernames(self):
         with self.assertRaises(AuthenticationFailed):
             JwtAuthentication().authenticate_credentials({'email': 'test@example.com'})
 
-    def test_authenticate(self):
-        """ Verify exceptions raised during authentication are properly logged. """
-        request = RequestFactory().get('/')
+    _MOCK_ANONYMOUS_RETURN = None
+    _MOCK_USER_AUTH_RETURN = ('mock-user', "mock-auth")
 
-        with mock.patch.object(JSONWebTokenAuthentication, 'authenticate', side_effect=Exception):
-            with mock.patch.object(Logger, 'debug') as logger:
-                self.assertRaises(
-                    Exception,
-                    JwtAuthentication().authenticate,
-                    request
-                )
-                self.assertTrue(logger.called)
+    @ddt.data(
+        ('get', True, True, _MOCK_ANONYMOUS_RETURN),
+        ('get', True, True, _MOCK_USER_AUTH_RETURN),
```

Comment:
The 2 above cases have the same input values, but different expected outputs. ?

Reply:
Ah. I see that the last value is the mocked return value instead of the expected result.
Suggestion: `expected_user_and_auth` -> `mocked_return_value_user_and_auth`

Reply:
Updated tests as discussed.  Also, removed 'get' tests, because I think that added unnecessary complication.